doc/md/guides/orms/hibernate.mdx

```diff
@@ -0,0 +1,439 @@
+
+In this guide, we will use the [Hibernate Atlas Provider](https://github.com/ariga/atlas-provider-hibernate)
+to automatically plan schema migrations for a Hibernate project.
+
+### Installation
+
+Install Atlas from macOS or Linux by running:
+```bash
+curl -sSf https://atlasgo.sh | sh
+```
+See [atlasgo.io](https://atlasgo.io/getting-started#installation) for more installation options.
+
+Install the provider by adding this plugin to your gradle project:
+
+<Tabs>
+<TabItem value="Gradle Kotlin" label="Gradle Kotlin" default>
+
+```gradle
+plugins {
+    id("io.atlasgo.hibernate-provider") version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Gradle Groovy" label="Gradle Groovy" default>
+
+```gradle
+plugins {
+    id "io.atlasgo.hibernate-provider" version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Maven" label="Maven">
+
+```
+Coming soon!
+```
+</TabItem>
+</Tabs>
+
+Verify that the plugin is installed, run this command
+
+<Tabs>
+<TabItem value="Gradle" label="Gradle" default>
+
+```bash
+./gradlew help --task schema
+```
+</TabItem>
+<TabItem value="Maven" label="Maven" default>
+
+```
+Coming soon!
+```
+</TabItem>
+</Tabs>
+
+### Library vs Gradle task
+
+The Atlas Hibernate Provider can be used in two modes:
+* **Gradle Task** - In most cases, you can use the provider as an gradle task to output
+the Hibernate schema and load the schema into Atlas.
+* **Library** - For other case, if you are initializing Hibernate in a custom way,
+using an older version of Hibernate or if you are not using gradle,
+you can use the provider as a library and create a small application that prints the schema to stdout.
+
+### Gradle Task
+
+In your project directory, create a new file named `atlas.hcl` with the following contents:
+
+```hcl
+data "external_schema" "hibernate" {
+  program = [
+    "./gradlew",
+    "-q",
+    "schema",
+    "--properties", "schema-export.properties"
+  ]
+}
+```
+
+And depending on your database, add an environment configuration. For example, for MySQL and PostgreSQL:
+
+<Tabs>
+<TabItem value="MySQL" label="MySQL" default>
+
+```hcl
+env "hibernate" {
+  src = data.external_schema.hibernate.url
+  dev = "docker://mysql/8/dev"
+  migration {
+    dir = "file://migrations"
+  }
+  format {
+    migrate {
+      diff = "{{ sql . \"  \" }}"
+    }
+  }
+}
+```
+</TabItem>
+<TabItem value="PostgreSQL" label="PostgreSQL">
+
+```hcl
+env "hibernate" {
+  src = data.external_schema.hibernate.url
+  dev = "docker://postgres/15/dev?search_path=public"
+  migration {
+    dir = "file://migrations"
+  }
+  format {
+    migrate {
+      diff = "{{ sql . \"  \" }}"
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+We need to let Atlas and the provider know which SQL dialect should be used. The dialect is specified in the
+properties file and in the dev database that Atlas will use.
+
+Create a resource named `schema-export.propertie` in `src/main/resource`, it should be part of the classpath by default.
```

Comment:
just double checking because it looks weird to me, is it supposed to be '.properties'? if so, you are missing an 's'